web/src/components/SpeakerCard.vue

```diff
@@ -86,7 +80,19 @@ export default {
       }
     },
   },
+  mounted() {
+    this.$getSpeakerTopics(this.speaker.id, this.setTopics, this.setError);
+  },
```

Comment:
So, instead of fetching 1~5 topics for each speaker (and probably fetching repeated ones several times across the list), for locations I put the [fetch of all locations on the `FindSpeaker` view](https://github.com/WWCodeTokyo/speak-her-db/blob/master/web/src/views/FindSpeaker.vue#L68) and passed the in-memory list [as props to the speaker card](https://github.com/WWCodeTokyo/speak-her-db/blob/master/web/src/views/FindSpeaker.vue#L31). How about we do the same for topics?

Reply:
And then merge them in the javascript with the foreign keys in the speaker? Sure, I could do that. Thought it might be more elegant to compose the right query from the backend, using caching like your example.